src/gui/tray/UnifiedSearchResultItem.qml

```diff
@@ -0,0 +1,232 @@
+import QtQml 2.12
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import Style 1.0
+import QtGraphicalEffects 1.0
+
+MouseArea {
+    id: unifiedSearchResultMouseArea
+
+    property int defaultHeight: 0
+
+    readonly property int iconLeftMargin: 12
+    readonly property int textLeftMargin: 4
+    readonly property int textRightMargin: 16
+    readonly property int categorySeparatorLeftMargin: 16
+
+    readonly property bool isFetchMoreTrigger: model.typeAsString === "FetchMoreTrigger"
+    readonly property bool isCategorySeparator: model.typeAsString === "CategorySeparator"
+    readonly property bool isDefaultItem: model.typeAsString === "Default"
+
+    property bool isFetchMoreInProgress: unifiedSearchResultsModel.currentFetchMoreInProgressProviderId === model.providerId
+    property bool isSearchInProgress: false
+
+    enabled: !isCategorySeparator && (!isFetchMoreTrigger || !isSearchInProgress)
+    hoverEnabled: !isCategorySeparator && (!isFetchMoreTrigger || !isSearchInProgress)
+
+    height: !isCategorySeparator ? defaultHeight : defaultHeight / 2
```

Comment:
setting `height` should be done at places of use not at place of definition
if there is a natural size based on content we have to use `implicitHeight` but in our case we really should define `height` at places of use